roles/network_plugin/flannel/tasks/main.yml

```diff
@@ -9,3 +9,19 @@
   register: flannel_node_manifests
   when:
     - inventory_hostname in groups['kube-master']
+
+- name: Flannel | Set CNI directory permissions
```

Comment:
This is the same as roles/kubernetes-apps/network_plugin/cni/tasks/main.yml could we re-use that role instead?

Reply:
Truth to be told, this part is duplicated in pretty much every CNI (macvlan, cilium, contiv, kube-router, weave, cloud)..
But we could indeed refactor that

Reply:
OK, let's refactor into a later PR then.